File: class25/python/AP2502-Exemplo02.py
# SimuladorEmprestimo — Aula 25
# Simulador de empréstimo com cálculo de parcelas

valor = float(input("Valor do empréstimo: R$ "))
taxa = float(input("Taxa mensal (%): "))
meses = int(input("Número de parcelas: "))
taxa_decimal = taxa / 100
parcela = valor * (taxa_decimal * (1 + taxa_decimal)**meses) / ((1 + taxa_decimal)**meses - 1)
total = parcela * meses
print(f"\nParcela mensal: R$ {parcela:.2f}")
print(f"Total a pagar: R$ {total:.2f}")
print(f"Juros totais:  R$ {total - valor:.2f}")

File: class25/java/AP2502-Exemplo02.java
import java.util.Scanner;
public class SimuladorEmprestimo {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Valor do empréstimo: R$ ");
        double valor = sc.nextDouble();
        System.out.print("Taxa mensal (%): ");
        double taxa = sc.nextDouble()/100;
        System.out.print("Número de parcelas: ");
        int meses = sc.nextInt();
        double parcela = valor * taxa * Math.pow(1+taxa, meses) / (Math.pow(1+taxa, meses)-1);
        double total = parcela * meses;
        System.out.printf("\nParcela mensal: R$ %.2f%n", parcela);
        System.out.printf("Total a pagar: R$ %.2f%n", total);
        System.out.printf("Juros totais:  R$ %.2f%n", total-valor);
        sc.close();
    }
}